Sources/OTel/OTelCore/Logging/Processing/Batch/OTelBatchLogRecordProcessor.swift

```diff
@@ -66,19 +66,27 @@ actor OTelBatchLogRecordProcessor<Exporter: OTelLogRecordExporter, Clock: _Concu
         let mergedSequence = merge(timerSequence, explicitTickStream).cancelOnGracefulShutdown()
 
         await withTaskGroup { taskGroup in
-            taskGroup.addTask {
-                for await log in self.logStream.cancelOnGracefulShutdown() {
-                    await self._onLog(log)
+            await withGracefulShutdownHandler {
+                taskGroup.addTask {
+                    for await log in self.logStream {
+                        self.logger.debug("Consuming log from stream")
```

Comment:
Do we want these to be `trace` instead of `debug`? `debug` seems a bit excessive to me given that we'd log both of these statements for every log record.